pkg/tfbridge/provider_test.go

```diff
@@ -5113,7 +5113,8 @@ func TestPlanResourceChangeUnknowns(t *testing.T) {
 				"setBlockProps":[],
 				"listBlockProps":[],
 				"nestedListBlockProps":[],
-				"maxItemsOneBlockProp":null
+				"maxItemsOneBlockProp":null,
+				"__meta":"{\"_new_extra_shim\":{}}"
```

Comment:
I wonder if this will cause pain somewhere else? I.e. re-recording of upgrade tests?